src/app/core/components/MobileMenu/MobileWalletDisplay.tsx

```diff
@@ -68,8 +66,6 @@ function MobileWalletDisplay({
                   <MobileWalletDisconnectButton
                     handleDisconnect={() => disconnectAsync()}
                   />
-
-                  {/* <button onClick={addToken}>add token to wallet</button> */}
```

Comment:
curious about this removal, will it need to be added back later? or is it just safe to remove?

Reply:
There should be an add token button but I'm thinking this is just dead code right now. Should be a separate issue to add the button with styling.